app/views/lists/show.html.erb

```diff
@@ -6,7 +6,12 @@
     <br>
     <%= @list.content %>
     <br>
+    <% @list.entries.each do |entry| %>
```

Comment:
Grrrrrr, what is [Issue 27's](https://github.com/Nakirendral/ListApp/issues/27) feature work doing here?